packages/server/src/utilities/accessLevels.js

```diff
@@ -0,0 +1,79 @@
+const viewController = require("../api/controllers/view")
+const modelController = require("../api/controllers/model")
+const workflowController = require("../api/controllers/workflow")
+
+exports.ADMIN_LEVEL_ID = "ADMIN"
+exports.POWERUSER_LEVEL_ID = "POWER_USER"
+
+exports.READ_MODEL = "read-model"
+exports.WRITE_MODEL = "write-model"
+exports.READ_VIEW = "read-view"
+exports.EXECUTE_WORKFLOW = "execute-workflow"
+exports.USER_MANAGEMENT = "user-management"
+exports.BUILDER = "builder"
+exports.LIST_USERS = "list-users"
+
+exports.adminPermissions = [
+  {
+    name: exports.USER_MANAGEMENT,
+  },
+]
+
+exports.generateAdminPermissions = async instanceId => [
+  ...exports.adminPermissions,
+  ...(await exports.generatePowerUserPermissions(instanceId)),
```

Comment:
It's weird to call functions on the `exports` object in node. Define you functions as normal and export them at the bottom:

```
module.exports = {
  generateAdminPermissions
}
```
etc